ONSdigital/sdc-global-design-patterns
# Improve deploy mechanism

### Changes

Simplifies deployment to Github Pages by removing the need to have to build and commit changes before deployment.

New deployment process is as simple as running `yarn deploy`—this will run the build task and then deploy to Github Pages in one action.

### How to test

1. remove local /dist folder
2. `yarn deploy`
3. Check changes are deployed
